rasa/core/policies/ensemble.py

```diff
@@ -790,20 +721,18 @@ def _check_policy_for_forms_available(
     if not ensemble:
         return
 
-    from rasa.core.policies.form_policy import FormPolicy
-
-    suited_policies_for_forms = (FormPolicy, RulePolicy)
+    suited_policies_for_forms = (RulePolicy)
```

Comment:
These parenthesis are doing nothing. You would need a comma to make it a tuple, but probably a tuple is not needed now.

Reply:
Removed the tuple and just now check that there is a rule policy instance in the config to handle forms.